elementary/monitor/api/tests/schema.py

```diff
@@ -53,11 +83,8 @@ def format_time_utc(cls, time_utc):
         return convert_partial_iso_format_to_full_iso_format(time_utc)
 
 
-class TotalsInvocationsSchema(BaseModel):
-    errors: Optional[int] = 0
-    warnings: Optional[int] = 0
-    passed: Optional[int] = 0
-    resolved: Optional[int] = 0
+class TotalsInvocationsSchema(TotalsSchema):
+    pass
```

Comment:
Why do we need both `TotalsInvocationsSchema` and `TotalsSchema`?

Reply:
Well they used to be different and I somehow like that we define a schema for each data type (even if they are the same).
But I agree that currently its a bit redundant